src/HatchwaysBlog.test.js

```diff
@@ -76,6 +76,28 @@ test("Changing page size from 15 to 25 should render 25 posts", async () => {
   }
 });
 
+test("Page 1 is initially selected", async () => {
```

Comment:
Test `aria-current` here, so that candidates do not get surprised by private tests.